zenoh/src/api/builders/sample.rs

```diff
@@ -99,6 +93,14 @@ impl SampleBuilder<SampleBuilderPut> {
             _t: PhantomData::<SampleBuilderPut>,
         }
     }
+
+    pub fn payload<IntoZBytes>(mut self, payload: IntoZBytes) -> Self
```

Comment:
I don't think this is needed. Actual payload is already provided in the `put()` function.

Reply:
You are right. In that case it makes sense to keep it.